Makefile

```diff
@@ -131,9 +131,32 @@ up:
 down:
 	$(DOCKER_COMPOSE) down -v
 
-.PHONY: upgrade
-upgrade: node_modules cache assets front-packages javascript-prod css javascript-extensions
+.PHONY: ifixit-upgrade
+ifixit-upgrade: dependencies cache assets front-packages javascript-prod css javascript-extensions
 	bash vendor/akeneo/pim-community-dev/std-build/install-required-files.sh
 	patch -p0 < patches/migrations.patch
 	cp .env .env.upgrade
 	cp .env.local .env
+	# Some migrations need elasticsearch to be running
```

Comment:
I'm not sure which boot script you're referring to in this commit 2fe044f4c4e0dac8f987bef3269d48982d0f3179

Reply:
I apparently only linked it in one direction!

https://github.com/iFixit/server-templates/pull/4005/commits/8e7028124585906cfa7343331cb514e657826d60